specification/attributes/column_naming_convention.md

```diff
@@ -18,16 +18,17 @@ Naming convention for columns appearing in billing data.
 
 ## Requirements
 
-* All columns defined by FOCUS MUST follow the following rules:
+* All columns defined by FOCUS(*) MUST follow the following rules:
```

Comment:
@SonalGarg-SG I assume the (*) was added by accident? If not necessary, can you quickly commit this and we can push this through. If needed, can you explain what it's for?

```suggestion
* All columns defined by FOCUS MUST follow the following rules:
```

Reply:
@udam-f2  I added that as Michael request the "MUST" records be moved ahead of the "SHOULD". 
But there was a reference to the Focus column's in the records so I decided to put an asterisk just to show where the reference was from. If it seems unnecessary, it can be removed.